### PR TITLE
Actions Refactor

### PR DESCRIPTION
**Pull request by mxcl**

### CODE REVIEW

**Comment by mxcl** [comment]
I think perhaps I went too far or not far enough. LOL. LMK what you think.

**Comment by mxcl** [comment]
all `@cache-brewkit` need to be `@main` before merging.

I don’t want to version this repo since it's really about the formula and that just adds headache.

**Comment by mxcl** [comment]
Honestly it all still feels like a massive mess

**Comment by mxcl** [comment]
The CI failure should be fixed in 0.24.4

**Comment by mxcl** [comment]
@jhheider I can't figure out how to merge this, up to you I’m afraid

**Comment by jhheider** [comment]
I can rebase (I've been keeping up with the changes in `main`). but I haven't worked out the hashFiles() returning nothing yet. :/

File: .github/actions/cache/action.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ inputs:
   cache-hash:
     description: hash of the cache to use
     required: true
-    default: ${{ hashFiles('./deno.jsonc') }}
+    default: ${{ hashFiles('**/deno.jsonc') }}
```
Comment (jhheider) [inline .github/actions/cache/action.yml:8]: yes, at worst it invalidates too aggressively.

**Comment by mxcl** [comment]
Green again. I’m personally scared tho.

I suggest you merge (if you still want to) when ready and with a fresh day ahead of you.

**Comment by mxcl** [comment]
After merge all the `@cache-brewkit` will need changing tho.

**Comment by jhheider** [comment]
Yeah, I'll either do it early tomorrow, or late friday and deal with the fallout on the weekend.

**Pull request merged**